docs/source/mayavi/installation.rst

```diff
@@ -142,6 +142,58 @@ After installing VTK Python, you can install mayavi using `pip`::
 .. _PyPA: https://packaging.python.org/en/latest/current/
 .. _PyPI: https://pypi.python.org/pypi
 
+.. _install-with-edm:
+
+Installing with `Enthought Deployment Manager(EDM)`
+.....................
+
+EDM is Enthought's freely available python environment manager. It allows new python users to create light-weight and custom python environments. A major advantage of EDM is its robust state-of-the-art management of package dependencies, with support for resolving conflicts and ensuring consistent package version sets. It can be downloaded from 
+`here <https://www.enthought.com/products/edm/installers/>`_.
+
+EDM provides Python 2.7.x as well as Python 3.x runtimes. Python 2.7 is the default version. After installing EDM, mayavi can be easily installed from the terminal by using `edm`:: 
+
+  $ edm install mayavi pyqt
+  
+This will install mayavi along with 'pyqt'. After this, you can start a shell with the default python version by running::
+
+  $ edm shell
+  
+This should set Mayavi for the default python 2.7. 
+
+For python 3.6.x, you can install it by running: 
+
+  $ edm environments create --version 3.6 py3
+
+Where 'py3' is the name of the environment. You may change this name to something else if you wish. Once you create the environment, you can activate it as follows::
+
+  $ edm shell -e py3
+  
+You can now install mayavi as follows::
+
+  $ edm install mayavi pyqt
+  
+Note that 'pyqt' needs to be installed explicitly for python 3.6 as well as for python 2.7. 
+
+.. _installing-with-conda
+
+Installing from `Conda`
+.....................
+
+Conda is an open source package management environment management system for installing multiple versions of software packages and their dependencies. Conda is included in Anaconda and Miniconda.
+
+Miniconda is a small "bootstrap" version that includes conda, pythona, and the packages they depend on. You can install miniconda from here_.
+
+.. _here: https://conda.io/miniconda.html
```

Comment:
Sir, I have corrected the underlining mistakes. In conda section, I have included the installer links but due to same names to access the links for python 3.6 and 2.7, it is giving a warning. Should I change that? And should I do something similar for edm?